ref/appliance-command-reference/ams.md

```diff
@@ -0,0 +1,21 @@
+The `ams` command provides access to the Anbox Management Service (AMS) through subcommands. It exposes the AMS HTTPS service on the public endpoint of the machine on which the appliance is running. Using this command, you can [control AMS remotely](https://discourse.ubuntu.com/t/managing-ams-access/17774).
+
+## Usage
+
+    anbox-cloud-appliance ams <subcommand>
+
+## Subcommands
+
+### `expose`
+Enables access to AMS in the load balancer.
+
+#### Usage
```

Comment:
hmm, do we really need four Heading levels in this simple page?

Reply:
I felt that creating separate pages for subcommands would be overkill. So for being consistent across all topics, I have used the same heading format in all command reference topics.

Reply:
I kinda agree with Gary here: do we need that `Usage` heading? I feel like we could remove it everywhere.

Reply:
I have removed the `Usage` heading everywhere, does it read better now?

Reply:
I think it reads much better, thanks!